src/server/import-controller.js

```diff
@@ -22,24 +30,30 @@ class ImportController {
   }
 
   registerApi(router) {
-    router.get('/api/v1/imports', [needsPermission(permissions.MANAGE_IMPORT)], async (req, res) => {
+    router.get('/api/v1/imports', [needsPermission(permissions.MANAGE_IMPORT), validateQuery(getImportsQuerySchema)], async (req, res) => {
       const { importSourceName } = req.query;
 
-      if (!importSourceName) {
-        res.status(400).send('importSourceName query param is required');
-        return;
-      }
-
       const importSource = this.serverConfig.importSources.find(source => source.name === importSourceName);
-
       if (!importSource) {
-        res.status(400).send(`import source ${importSourceName} is unknown`);
-        return;
+        throw new NotFound(`${importSourceName} is not a known import source`);
```

Comment:
Should this rather be BadRequest? Though I would see the arguments for both options 😊

Reply:
I considered `importSourceName` an ID, so I thought, if that ID is not in the list of available sources, it could be NotFound, but BadRequest would also work in this scenario, I guess. I don't have a strong opinion on it.

Reply:
Ok then let's keep NotFound.